### ISSUE TITLE
Unused cap_drop expose in container shema

### ISSUE THREAD

**Issue by docjyJ** [nextcloud/all-in-one#5569]
### Steps to reproduce
1. In file, https://github.com/nextcloud/all-in-one/blob/main/php/containers-schema.json: `ctrl+f` > `cap_drop` or `expose`
2. In file, https://github.com/nextcloud/all-in-one/blob/main/php/src/ContainerDefinitionFetcher.php: `ctrl+f` > `cap_drop` or `expose`
3. They are defined but never used...

#### Other valuable info 

*(I'm rewriting mastercontainer in rust (to learn rust) and I found this...)*

**Comment by docjyJ** [comment]
Maybe this behavior is intended for yaml, but it's confusing.

**Comment by szaimen** [comment]
Yes, it is intended because we generate https://github.com/nextcloud/all-in-one/blob/main/manual-install/latest.yml out of it. Do you have a better idea how to do that?

**Comment by docjyJ** [comment]
Why have a cap_drop in YAML, but not with the mastercontainer ?

**Comment by szaimen** [comment]
Because cap_drop is by default applied for all containers: https://github.com/nextcloud/all-in-one/blob/267aebf941883bc21437e51974d14079aada1bc8/php/src/Docker/DockerActionManager.php#L519

**Comment by szaimen** [comment]
Or WDYM?

**Comment by szaimen** [comment]
Linking to https://github.com/nextcloud/all-in-one/pull/3377 which was introduced first. Afterwards https://github.com/nextcloud/all-in-one/pull/4116 was added

**Comment by szaimen** [comment]
`cap_drop`, `profiles`, `user`, `networks` or `expose`

Actually, all of them are only there for the docker-compose yaml and the helm chart because they are generated out of the containers.json. In the mastercontainer itself is a separate logic to handle these, thus they are not actually needed for the mastercontainer.

How should we proceed with this issue?

**Comment by docjyJ** [comment]
Move this to discussions as ideas?

**Comment by szaimen** [comment]
> Move this to discussions as ideas?

Sounds good 👍
